### DISCUSSION TITLE
Unclear description in official docs creating spy: 'ee.on' ? 'some event' ?

### DISCUSSION THREAD

**Discussion by Dmytro2V** [chaijs/chai#1524] [Support] [answered]
https://www.chaijs.com/plugins/chai-spies/

What is ee.on ?
and what is 'some event'?

"Creating Spies
In this module, a spy is either an empty function, or a wrapped named function. Once chai has been extended, you can create a spy through chai’s own interface.

function original () {
  // do something cool
}

const spy = chai.spy(original);

// then use in place of original
ee.on('some event', spy);

// or use without original
const spyAgain = chai.spy();
ee.on('some other event', spyAgain);"

**Comment by keithamus** [comment, marked as answer]
The code demonstrates an example of how to use the spy like a function.

In the example `ee` is assumed to be an instance of Node's `EventEmitter`. `'some event'` is an example event name. The point of the example is to show that `spy` can be passed around like a regular function.
